summary/types/glucose.go

```diff
@@ -237,7 +244,9 @@ func (b *GlucoseBucket) Update(r data.Datum, lastData *time.Time) (bool, error)
 		return false, nil
 	}
 
-	b.GlucoseRanges.Update(record)
+	if err := b.GlucoseRanges.Update(record); err != nil {
+		return false, err
```

Comment:
Does this mean that if a single value has an error then it will cause the entire summary to fail?